app/react/Documents/components/specs/DocumentSidePanel.spec.js

```diff
@@ -185,5 +208,10 @@ describe('DocumentSidePanel', () => {
       const ownProps = {};
       expect(mapStateToProps(state, ownProps).connectionsGroups).toBe('connectionsGroups');
     });
+
+    it('should map default language', () => {
+      const ownProps = {};
+      expect(mapStateToProps(state, ownProps).defaultLanguage).toBe('es');
```

Comment:
As mentioned before, this is:
- the wrong wording, this is currentLanguage, not defaultLanguage
- I still can't see where this prop is actually used